examples/acquisition_settings/demo_acquisition_settings.yaml

```diff
@@ -66,3 +66,57 @@ ls_microscope_settings:
     - ['Camera', 'BitDepth', '12']
   z_sequencing_settings:
     - ['Z', 'UseSequences', 'Yes']
+
+# Zarr output settings for the label-free acquisition (demo)
+lf_zarr_settings:
+  data_type: 'UINT16'
+  multiscale: false
+  max_threads: 2  # limit threads for demo
+  chunk_sizes:  # unit: pixels per chunk
+    x: 64
+    y: 64
+    z: 1
+    c: 1
+    p: 1
+    t: 1
+  shard_sizes:  # unit: chunks per shard
```

Comment:
in iohub we use the term `shards_ratio` which I think is also what you mean here. Could we use the same terminology in shrimpy and iohub, even if that's different from zarr-python or acquire-zarr. I suggest renaming `chunk_sizes` to `chunks` and `shard_sizes` to `shards_ratio`. https://github.com/czbiohub-sf/iohub/blob/56a818632727d6291d061f98562c392b6c8ca841/iohub/ngff/nodes.py#L783